src/renderer/components/data-settings/data-settings.js

```diff
@@ -125,9 +125,43 @@ export default Vue.extend({
           let textDecode = new TextDecoder('utf-8').decode(data)
           textDecode = textDecode.split('\n')
           textDecode.pop()
+          textDecode = textDecode.map(data => JSON.parse(data))
+
+          // TODO: Move this function to a better place
+          function convertOldFreeTubeFormatToNew(oldData) {
+            const convertedData = []
+            for (const channel of oldData) {
+              for (const profile of channel.profile) {
+                let index = convertedData.findIndex(p => p.name === profile.value)
+                if (index === -1) { // profile doesn't exist yet
+                  convertedData.push({
+                    name: profile.value,
+                    bgColor: '#000000',
```

Comment:
I understand defaulting this to black since the db file doesn't return a color.  Let's instead has it set to a random color.  I already have functions that can do this for you.  You will want `getRandomColor` and `calculateColorLuminance` from the utils VueX store to do this for you.  Let me know if you need help with the syntax as it might be strange to make these work if you're rather new to the framework.